docs/src/capi/tutorial/getting-started.md

```diff
@@ -94,7 +94,49 @@ export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-a
 ```
 
 You are now all set to deploy the AWS CAPI infrastructure provider.
```

Comment:
```suggestion
You are now all set to deploy the MAAS infrastructure provider.
```

Reply:
That is the tab of the AWS infrastructure provider. So, this should be correct. 
The MAAS one is below.